importer/make_KMB_info.py

```diff
@@ -595,8 +608,18 @@ def get_exact_cat_from_name(self, cache):
         """
         exact_match = False
         exact_category_from_name = self.kmb_info.category_exists(self.namn, cache)
-        if exact_category_from_name:
-            exact_category_from_name = pywikibot.Page(self.commons, self.namn)
+
+        if exact_category_from_name is False:
```

Comment:
Don't think you want this. even if `exact_category_from_name is False` we probably want to check for a church.

How about:

```python
exact_match = False

# churches are done separately
if ("Religionsutövning - kyrkor" in self.item_classes):
    if self.get_exact_match_church():
        return

# Not a church, more generalised guesswork
exact_category_from_name = self.kmb_info.category_exists(self.namn, cache)
if exact_category_from_name:
    exact_category_from_name = pywikibot.Page(self.commons, self.namn)
    parent_cats = exact_category_from_name.categories()
    ...
```
the end of the function can then be left as it was before this PR.

Reply:
Done, I think. Waiting for new church cat list to test.